library/src/main/java/com/chuckerteam/chucker/internal/ui/transaction/TransactionPayloadFragment.kt

```diff
@@ -158,6 +228,47 @@ internal class TransactionPayloadFragment :
         }
     }
 
+    private class FileSaverTask(val fragment: TransactionPayloadFragment) :
+        AsyncTask<Triple<Int, Uri, HttpTransaction>, Unit, Boolean>() {
+
+        @Suppress("NestedBlockDepth")
+        override fun doInBackground(vararg params: Triple<Int, Uri, HttpTransaction>): Boolean {
+            val (type, uri, transaction) = params[0]
+            try {
+                val context = fragment.context ?: return false
+                context.contentResolver.openFileDescriptor(uri, "w")?.use {
+                    FileOutputStream(it.fileDescriptor).use { fos ->
+                        when {
+                            type == TYPE_REQUEST ->
+                                transaction.requestBody?.byteInputStream()?.copyTo(fos)
+                            transaction.responseBody != null ->
```

Comment:
I suggest to keep all condition checks in braces, as well as add curly braces for outcomes, because it is hard to read in such cases.